### DISCUSSION TITLE
Nth row customization and spacing

### DISCUSSION THREAD

**Discussion by carnatak** [lvjr/tabularray#281] [Q&A]
Firstly, LOVE tabularray. Made my code so much simpler. 
I would like to know if there's a way to customize the "Nth" row (in my case every 3rd row) in a tabularray. 
row{even} does this for even rows, but I want to customize it further. 

```latex
\documentclass{article}

\usepackage{xcolor}
\usepackage{tabularray}

\begin{document}

\begin{tblr}{row{even} = {font = \scriptsize ,fg = blue2}}
\hline
  Alpha   & Beta  & Gamma  \\
\hline
  Epsilon & Zeta  & Eta    \\
% I want to make every third row have a smaller font \scriptsize and have a smaller height
\hline
  Iota    & Kappa & Lambda \\
\hline
\end{tblr}

\end{document}
```

**Comment by lvjr** [comment]
You may see https://github.com/lvjr/tabularray/issues/278  for an example of defining a new selector with `\NewChildSelector` command.

**Reply by carnatak** [reply]
Thank you very much @lvjr ! This example certainly looks relevant. 
Could you please elaborate on what this piece of code is specifying? I'm fairly new to Tex, so would appreciate an explanation. 

```
\ExplSyntaxOn
\NewChildSelector{eachthirdtwo}{
    \int_step_inline:nnnn {2} {3} {\l_tblr_childs_total_tl}
        {\clist_put_right:Nn \l_tblr_childs_clist {##1}}
}
\ExplSyntaxOff
```

**Reply by lvjr** [reply]
This is latex3 code. As a new user, you only need to know it selects all rows/columns/hlines/vlines if their numbers are of the type `2+3x`. For example, if the table has 15 rows, `row{eachthirdtwo}` select rows `2, 5, 8, 11, 14`.

**Reply by carnatak** [reply]
Thank you! The ```2+3x``` explanation is perfect.
